x-pack/plugins/observability_solution/infra/public/alerting/metric_threshold/components/expression_row.tsx

```diff
@@ -200,7 +200,7 @@ export const ExpressionRow = ({
 
   return (
     <>
-      <EuiFlexGroup gutterSize="xs">
+      <EuiFlexGroup gutterSize="xs" data-test-subj="expressionRow">
```

Comment:
nit: It may be good to give a slightly more descriptive name to this test ID; it's unique to Kibana at the moment but maybe `metricThresholdExpressionRow` or something along those lines would be better. Not a show-stopper.

Reply:
Good idea, fixed in [c9146b6](https://github.com/elastic/kibana/pull/192736/commits/c9146b64a36b515b09da8e98044721d229740dd5)